sdks/java/core/src/test/java/org/apache/beam/sdk/testing/PAssertTest.java

```diff
@@ -378,7 +379,7 @@ public void testContainsInAnyOrderFalse() throws Exception {
             + " but the message was \""
             + exc.getMessage()
             + "\"",
-        expectedPattern.matcher(exc.getMessage()).find());
+        expectedPattern.matcher(Throwables.getStackTraceAsString(exc)).find());
```

Comment:
Can this be tightened? I'd like for the case where I don't provide an auxiliary message to be unchanged, and for the new case to have the detailed cause immediately nested.